### PR TITLE
Enable interpreter properties edit

### PR DESCRIPTION
**Pull request by minahlee**
This PR enables you to edit interpreter setting properties.

### CODE REVIEW

**Comment by swkimme** [comment]
Great job!

BTW, it will create big conflict with my PR. 
https://github.com/NFLabs/zeppelin/pull/321

I'll need to manually merge it. 
T_T

File: zeppelin-server/src/main/java/com/nflabs/zeppelin/rest/InterpreterRestApi.java [hunk 1]
```diff
@@ -87,6 +87,23 @@ public Response newSettings(String message) throws InterpreterException, IOExcep
...
   }
 
+  @PUT
+  @Path("setting/{settingId}")
+  public Response updateSetting(String message, @PathParam("settingId") String settingId) {
+    logger.info("Update interpreterSetting {}", settingId);
+    try {
+      Properties p = gson.fromJson(message, Properties.class);
+      interpreterFactory.setPropertyAndRestart(settingId, p);
+    } catch (InterpreterException e) {
```
Comment (anthonycorbacho) [inline zeppelin-server/src/main/java/com/nflabs/zeppelin/rest/InterpreterRestApi.java]: Maybe it is good to start to handle different error, for exemple the `JsonSyntaxException`, imagine in my request i send  ``Yo died``, then i will get a 500, maybe we can catch this and return a proper http code like `400 Bad Request`

**Comment by swkimme** [comment]
Tested, works great!
Also manually merged into #321

**Comment by minahlee** [comment]
Thanks for taking care of the conflict!

**Comment by Leemoonsoo** [comment]
@minahlee This is really useful improvements. Great work.
But it still have conflict. Can you resolve it?

**Comment by Leemoonsoo** [comment]
Tested and Looks good to me!

**Pull request merged**